gui/biomes.lua

```diff
@@ -0,0 +1,392 @@
+-- Show biomes on the map.
+
+local RELOAD = false -- set to true to help with debugging
+
+local gui = require('gui')
+local widgets = require('gui.widgets')
+local guidm = require('gui.dwarfmode')
+
+local TILE_HIGHLIGHTED = 2585 -- yellow-ish indicator
```

Comment:
This needs to be looked up dynamically since its tilepos might change.
```lua
local TILE_HIGHLIGHTED = dfhack.textures.getOnOffTexposStart()
if TILE_HIGHLIGHTED < 0 then
    TILE_HIGHLIGHTED = --whatever the number for capital X is maybe
end
```

Reply:
is there a range where the textures are fixed? First 256? Also, what does switching to text mode do to this?

Reply:
I thought there was, but then https://github.com/DFHack/dfhack/issues/3262 was filed and now I don't trust even the initial "pre-world" textures.

For a little background, run `devel/tile-browser` on the title screen. you can see that only about 2500 textures are defined. Those are the ones that I thought were constant, but maybe not between Steam and Classic. The 2585 one is loaded by DFHack, and we might add more textures at any time, so it definitely cannot be treated as a constant.

Reply:
first 256 are *probably* constant, since those are the text glyphs.